packages/web3-eth-abi/types/index.d.ts

```diff
@@ -40,3 +40,6 @@ export class AbiCoder {
         topics: string[]
     ): { [key: string]: string };
 }
+
+declare var coder: AbiCoder;
```

Comment:
```suggestion
declare const coder: AbiCoder;
```
I think you need to use `const` here

Reply:
Why need? If it doesn't match you coding style, you can change it afterwards, or close and commit yourself. For me its only important that this bug gets fixed, not how.

Reply:
just fix it, please. and also check tests. They do not pass after your changes.